pincer/objects/guild/channel.py

```diff
@@ -779,6 +779,69 @@ async def fetch_message(self, message_id: int) -> UserMessage:
             await self._http.get(f"channels/{self.id}/messages/{message_id}")
         )
 
+    async def history(
+        self, limit: int = 50,
+        before: Union[int, Snowflake] = None,
+        after: Union[int, Snowflake] = None,
+        around: Union[int, Snowflake] = None,
+    ) -> AsyncIterator[UserMessage]:
+        """
+        Returns a list of messages in this channel.
+
+        Parameters
+        ----------
+        around
+            The message ID to look around.
+        after
+            The message ID to look after.
+        before
+            The message ID to look before.
+        limit
+            The maximum number of messages to return.
+
+        Returns
+        -------
+        AsyncIterator[:class:`~pincer.objects.message.user_message.UserMessage`]
+            An iterator of messages.
+        """
+
+        if isinstance(before, int):
+            before = Snowflake(before)
+
+        if isinstance(after, int):
+            after = Snowflake(after)
+
+        if isinstance(around, int):
+            around = Snowflake(around)
```

Comment:
Is this needed? Functionally, an `int` and a `Snowflake` act the same other than a few added methods, none of which that are used here.
```suggestion
```